include/checks.h

```diff
@@ -23,7 +23,6 @@
 
 #define CHECK_NOT_AUDITED_TLV_TAG 0x9F
 #define CHECK_NOT_AUDITED_TLV_VAL 0x01
-#define CHECK_NOT_AUDITED_MAX_LEN 0x40
```

Comment:
It was unused ?

Reply:
it was used in checks.c has an hardcoded max value for searching the pending ledger review tag in the install_parameters. I replaced it with the real length of the install_parameters.